src/main/java/org/alfresco/opencmis/CMISConnector.java

```diff
@@ -3183,28 +3185,34 @@ private void processSecondaryTypes(NodeRef nodeRef, List secondaryTypes, Map<Str
             }
         }
 
-        Set<QName> ignore = new HashSet<QName>();
-        ignore.add(ContentModel.ASPECT_REFERENCEABLE);
-        ignore.add(ContentModel.ASPECT_LOCALIZED); 
-        ignore.add(ContentModel.ASPECT_WORKING_COPY);
+        Set<QName> aspectsToIgnore = new HashSet<>();
+        aspectsToIgnore.add(ContentModel.ASPECT_REFERENCEABLE);
+        aspectsToIgnore.add(ContentModel.ASPECT_LOCALIZED);
+        aspectsToIgnore.add(ContentModel.ASPECT_WORKING_COPY);
+
+        Set<String> namespacesToIgnore = new HashSet<>(singletonList(NamespaceService.SYSTEM_MODEL_1_0_URI));
 
         // aspects to add == the list of secondary types - existing aspects - ignored aspects
         Set<QName> toAdd = new HashSet<QName>(secondaryTypeAspects);
         toAdd.removeAll(existingAspects);
-        toAdd.removeAll(ignore);
+        toAdd.removeAll(aspectsToIgnore);
+        toAdd.removeIf(a -> namespacesToIgnore.contains(a.getNamespaceURI()));
 
         // aspects to remove == existing aspects - secondary types
         Set<QName> aspectsToRemove = new HashSet<QName>();
         aspectsToRemove.addAll(existingAspects);
-        aspectsToRemove.removeAll(ignore);
+        aspectsToRemove.removeAll(aspectsToIgnore);
         Iterator<QName> it = aspectsToRemove.iterator();
         while(it.hasNext())
         {
             QName aspectQName = it.next();
             TypeDefinitionWrapper w = getOpenCMISDictionaryService().findNodeType(aspectQName);
-            if(w == null || secondaryTypeAspects.contains(aspectQName))
+            if(w == null || secondaryTypeAspects.contains(aspectQName) || namespacesToIgnore.contains(aspectQName.getNamespaceURI()))
             {
-                // the type is not exposed or is in the secondary types to set, so remove it from the to remove set
+                // the type is not exposed,

```

Comment:
Not sure why the comment has to be spread over four lines.

Reply:
On one line it goes to column 161, which seems a bit long for my liking